### ISSUE TITLE
execute run_pascal.sh

### ISSUE THREAD

**Issue by JiaojiaoYe1994** [weixuansun/wsss_sam#3]
Hi, I have met an issue when I try to run `bash run_pascal.sh`, the program return to debug mode in Pdb mode, any ideas about that? Thank you.

Part of shell output is like,
```
{'aeroplane': 1, 'bicycle': 2, 'bird': 3, 'boat': 4, 'bottle': 5, 'bus': 6, 'car': 7, 'cat': 8, 'chair': 9, 'cow': 10, 'diningtable': 11, 'dog': 12, 'horse': 13, 'motorbike': 14, 'person': 15, 'pottedplant': 16, 'sheep': 17, 'sofa': 18, 'train':19, 'tvmonitor': 20, 'bowl': 11, 'plate': 11, 'food': 11, 'fruit': 11, 'glass': 11, 'dishes': 11, 'halter': 13, 'saddle': 13, 'tv': 20, 'monitor': 20}
{'horse': ['halter', 'saddle'], 'diningtable': ['bowl', 'plate', 'food', 'fruit', 'glass', 'dishes'], 'tvmonitor': ['tv', 'monitor']}
/home/.conda/envs/fusion/lib/python3.7/site-packages/torch/functional.py:504: UserWarning: torch.meshgrid: in an upcoming release, it will be required to pass the indexing argument. (Triggered internally at ../aten/src/ATen/native/TensorShape.cpp:3190.)
  return _VF.meshgrid(tensors, **kwargs)  # type: ignore[attr-defined]
final text_encoder_type: bert-base-uncased
Some weights of the model checkpoint at bert-base-uncased were not used when initializing BertModel: ['cls.predictions.bias', 'cls.predictions.transform.dense.weight', 'cls.predictions.transform.LayerNorm.bias', 'cls.seq_relationship.weight', 'cls.predictions.transform.dense.bias', 'cls.seq_relationship.bias', 'cls.predictions.transform.LayerNorm.weight']
- This IS expected if you are initializing BertModel from the checkpoint of a model trained on another task or with another architecture (e.g. initializing a BertForSequenceClassification model from a BertForPreTraining model).
- This IS NOT expected if you are initializing BertModel from the checkpoint of a model that you expect to be exactly identical (initializing a BertForSequenceClassification model from a BertForSequenceClassification model).
34 2007_001149 -- pottedplant.sofa.tvmonitor.tv.monitor
/home/.conda/envs/fusion/lib/python3.7/site-packages/transformers/modeling_utils.py:885: FutureWarning: The `device` argument is deprecated and will be removed in v5 of Transformers.
  "The `device` argument is deprecated and will be removed in v5 of Transformers.", FutureWarning
/home/.conda/envs/fusion/lib/python3.7/site-packages/torch/utils/checkpoint.py:31: UserWarning: None of the inputs have requires_grad=True. Gradients will be None
  warnings.warn("None of the inputs have requires_grad=True. Gradients will be None")
> /mnt/nas/2024/wsss_sam/GroundingDINO/groundingdino/util/utils.py(638)get_phrases_from_posmap_2()
-> token_ids = [tokenized["input_ids"][i] for i in non_zero_idx]
(Pdb)

**Comment by weixuansun** [comment]
Hi, I think you may toggle any breakpoint() open?
